### ISSUE TITLE
Admin: Dynamic report generation not announced to screen reader user after they activate Get report button and a report appears below.

### ISSUE THREAD

**Issue by JohnHewittCriterion** [nciinc/fs-permit-platform#601]
## Notes
After a report is generated it displays below but nothing is spoken to the screen reader user. 
Speak an announcement using aria live regions or move focus using focus management. 

Examples:
http://pauljadam.com/demos/aria-alert-validation.html
http://pauljadam.com/demos/form-errors-top.html
http://pauljadam.com/demos/aria-describedby-validation.html

![image.png](https://images.zenhubusercontent.com/5913513251aa2c30577de73e/9f2aeebe-720d-449a-
994b-4a03d7c3d780)

Linkage: CTF-75

## Acceptance criteria
- [x] user can see that report is generated and focused

## Tasks
- [x] shift focus to report after creation so that screen reader announces it

## Definition of Done
- [x] Code unit / e2e tested and comply with QWASP
- [x] No new security vulnerabilities
- [x] Internal team code review
- [x] Documentation / readme.md / CircleCI updated
- [x] Docker updated
- [x] Compare finished design with mockup
- [ ] Usability tested
